executor/write.go

```diff
@@ -1892,3 +1901,16 @@ func (e *UpdateExec) Close() error {
 func (e *UpdateExec) Open(goCtx goctx.Context) error {
 	return e.SelectExec.Open(goCtx)
 }
+
+func updateTableDelta(ctx context.Context, tableInfo *model.TableInfo, row []types.Datum, delta int64, count int64) {
+	h := domain.GetDomain(ctx).StatsHandle()
+	if h == nil {
```

Comment:
When will the handle be nil?

Reply:
In the bootstrap stage, the handle is not initialized.